src/test/java/attendance/domain/CrewTest.java

```diff
@@ -0,0 +1,157 @@
+package attendance.domain;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.assertj.core.api.SoftAssertions.assertSoftly;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import attendance.fixture.CrewFixture;
+import attendance.fixture.CrewsFixture;
+import attendance.fixture.DateTimeFixture;
+
+class CrewTest {
+
+    private Crews crews;
+    private Crew crew;
+
+    @BeforeEach
+    void setup() {
+        crews = CrewsFixture.onlyPobi();
+        crew = CrewFixture.pobi();
+    }
+
+    @Test
+    @DisplayName("닉네임과 등교 시간을 입력하면 출석할 수 있다")
+    void attendanceTest() {
+        // when then
+        assertThatCode(() -> {
+            Crew found = crews.get("pobi");
+            found.attendance(DateTimeFixture.MONDAY, DateTimeFixture.MONDAY_START_TIME);
+        }).doesNotThrowAnyException();
+    }
+
+    @Test
+    @DisplayName("이미 출석한 경우 다시 출석을 시도하면 예외를 반환한다")
+    void attendanceExceptionTest() {
+        // given
+        crew.attendance(DateTimeFixture.MONDAY, DateTimeFixture.MONDAY_START_TIME);
+
+        // when then
+        assertThatThrownBy(() -> {
+            crew.attendance(DateTimeFixture.MONDAY, LocalTime.of(10, 00));
+        }).isInstanceOf(IllegalArgumentException.class);
```

Comment:
exception 을 테스트할 때에는 꼭 예외 메세지까지 테스트해주세요!

```suggestion
        }).isInstanceOf(IllegalArgumentException.class)
          .hasMessageContaining("이미 출석한 경우 다시 출석할 수 없습니다. 출석 수정 기능을 이용해 주세요.");
```

참고 : https://bottom-to-top.tistory.com/97

Reply:
넵 명심하겠습니다!
[refactor: 예외 메시지 관리를 위한 열거형 추가](https://github.com/woowacourse/java-attendance/pull/100/commits/e5e3d5b18026a2c38fa3491b75c080bc0510aab8)
[test: 예외 메시지 검증 로직 추가](https://github.com/woowacourse/java-attendance/pull/100/commits/d91e3d52fcb573e086b03f3f41d62155883df26e)